components/PromptRemove.vue

```diff
@@ -34,6 +34,10 @@ export default {
         return `resource${ this.toRemove.length === 1 ? '' : 's' }`;
       }
 
+      if (schema?.attributes?.kind === 'VirtualMachineImage') {
+        schema.attributes.kind = 'Image';
+      }
+
```

Comment:
Is there anything special here？

Reply:
We would like to use "Image" to instead of "VirtualMachineImage"